sdk/src/test/scala/io/horizen/account/sc2sc/AccountCrossChainRedeemMessageTest.scala

```diff
@@ -0,0 +1,33 @@
+package io.horizen.account.sc2sc
+
+import io.horizen.account.sc2sc.CrossChainRedeemMessageProcessorImpl.receiverSidechain
+import io.horizen.utils.BytesUtils
+import org.junit.Assert.assertTrue
+import org.junit.Test
+
+class AccountCrossChainRedeemMessageTest {
```

Comment:
If we want to produce a negative test we need to have wrong parameters for the encoder. This is gonna trigger the validator, right after the constructor, that is not injected since we don't allow the possibility to have different validators from the default one. So we should mock the static validator to bypass it and have a malformed byte array. But this is not gonna happen in the real scenario where the validator's always gonna run. So I think it's better not to add such test case